### PR TITLE
Discussion latest comment previews

### LINKED ISSUE #1426: Board list latest-post summary: show content snippet
On a given Talk's home page I think the content summaries of each board (showing the title of the board and the title of the most recently-updated thread) work really well. When the same summary is applied to each Talk sub-board's display, though, it means the title of each post is mentioned twice:

screen shot 2015-08-18 at 13 47 37

I think it would be more informative if the most-recently-updated line went 1 level deeper and displayed a snippet of the text (not including quoted lines that start with ">") of the most-recently-updated comment.

Apologies if this has been mentioned before.

### PR DESCRIPTION
**Pull request by aweiksnar**
- Adds comment plain-text content snippet to discussion previews
- Closes #1426

### CODE REVIEW

File: app/talk/latest-comment-link.cjsx [hunk 1]
```diff
@@ -5,8 +5,17 @@ Avatar = require '../partials/avatar'
 PromiseRenderer = require '../components/promise-renderer'
 {Link} = require 'react-router'
 merge = require 'lodash.merge'
+{Markdown} = require 'markdownz'
 
-PAGE_SIZE = require('./config.coffee').discussionPageSize
+PAGE_SIZE = require('./config').discussionPageSize
+
+truncate = (string, ending = '', length = 80) ->
+  string.trim().slice(0, (length - ending.length)) + ending
```
Comment (brian-c) [inline app/talk/latest-comment-link.cjsx]: Maybe don't append `ending` if `string.trim().length <= length`?

File: app/talk/latest-comment-link.cjsx [hunk 2]
```diff
@@ -5,8 +5,17 @@ Avatar = require '../partials/avatar'
...
+
+latestCommentText = (discussion) ->
+  container = document.createElement('span')
+  React.render(<Markdown content={discussion.latest_comment?.body} />, container)
+  container.textContent
```
Comment (brian-c) [inline app/talk/latest-comment-link.cjsx]: This is very clever. :+1:

File: css/talk.styl [hunk 3]
```diff
@@ -294,7 +306,7 @@ COPY_GREY_LIGHT = #afaeae
       &:hover
         color: COBOLT_BLUE
 
-    a:last-child
+    a:nth-child(3)
```
Comment (brian-c) [inline css/talk.styl]: Could we assign this a class?

File: app/talk/latest-comment-link.cjsx (hunk 1 above)
Comment (aweiksnar) [inline app/talk/latest-comment-link.cjsx]: Yeah good idea - just added in a484f16

File: css/talk.styl (hunk 3 above)
Comment (aweiksnar) [inline css/talk.styl]: Yeah i guess that's kinda cryptic - added in a484f16

File: app/talk/latest-comment-link.cjsx (hunk 2 above)
Comment (aweiksnar) [inline app/talk/latest-comment-link.cjsx]: Thanks, wasn't sure how else to un-parse markdown

**Comment by brian-c** [comment]
Looks good, thanks!

**Pull request merged**